test/integration/epacems_test.py

```diff
@@ -67,15 +67,25 @@ def test_epacems_subset_input_validation(epacems_year_and_state, epacems_parquet
             epacems(epacems_path=path, **combo)
 
 
-def test_epacems_parallel(pudl_settings_fixture, pudl_ds_kwargs):
+def test_epacems_parallel(pudl_settings_fixture, pudl_ds_kwargs, tmpdir_factory):
```

Comment:
what is this new arg?

Reply:
`tmpdir_factory` is a built-in pytest fixture which can create a (disposable) temporary directory. We needed a place to stash these (temporary) parquet outputs where they won't interfere with the real outputs and/or existing parquet datasets when the tests are run with `--live-dbs`